### ISSUE TITLE
Web support?

### ISSUE THREAD

**Issue by tomk9** [AlessioLuciani/flutter-pdf-text#9]
Is it possible to add web support using, for example, this package?
https://github.com/mozilla/pdf.js

**Comment by AlessioLuciani** [comment]
Yes, this is probably one of the next steps to accomplish for this package. Thank you @tomk9  for the suggestion! By the way, I cannot guarantee a deadline for this feature.

**Comment by HarvsG** [comment]
Another possibile package that could be used. https://gitlab.com/autokent/pdf-parse

**Comment by chrisDK1977** [comment]
Are there still any plans for web support?
